homeassistant/components/asuswrt/sensor.py

```diff
@@ -190,13 +239,11 @@ def __init__(
     ) -> None:
         """Initialize a AsusWrt sensor."""
         super().__init__(coordinator)
-        self.entity_description: AsusWrtSensorEntityDescription = description
+        self.entity_description = description
 
-        self._attr_name = f"{router.name} {description.name}"
-        if router.unique_id:
-            self._attr_unique_id = f"{DOMAIN} {router.unique_id} {description.name}"
-        else:
-            self._attr_unique_id = f"{DOMAIN} {self.name}"
+        self._attr_unique_id = (
+            f"{DOMAIN} {router.unique_id or DEFAULT_NAME} {description.key}"
```

Comment:
We should not use `DOMAIN` and `DEFAULT_NAME` as  part of `unique_id`. `f"{router.unique_id}_{description.key}"` is OK.

Reply:
It is possible that router `unique_id` is `None` because config flow allow creation of a single instance for Router that do not provide a mac address. For this reason we use 'DEFAULT_NAME'. Of course I can remove `DOMAIN`  from here and in the migration procedure.

Reply:
Maybe I could use `entry_id` for entity `unique_id`?

Reply:
Sorry, my mistake, I thought `router.unique_id` is `entry_id` now, not enough coffee. Of course we can use `entry_id` as `router.unique_id`.

Reply:
This should be ok now and migration to new unique id for existing entities is implemented.